pyproject.toml

```diff
@@ -0,0 +1,40 @@
+
+[project]
+name = "vcenter-operator"
+version = "0.1.0"
+description = ""
+authors = [
+    {name = "Fabian Wiesel", email = "fabian.wiesel@sap.com"},
+]
+dependencies = [
+    "attrs",
+    "dnspython @ git+https://github.com/sapcc/dnspython.git@ccloud",
+    "scrypt==0.8.6",
+    "pyvmomi",
+    "jinja2<4,>=3",
+    "kubernetes~=26.1",
+    "keystoneauth1",
+    "dumb-init"]
```

Comment:
Why didn't the `cryptography` and `scrypt` dependency make it here? Doesn't `vcenter_operator.masterpassword` still need them?

Reply:
Cryptography was only in a comment
```
# cryptography >= 1.6
``` 
as an alternative to `scrypt`, which is still there as the third dependency.

Reply:
Looks like I can't read. Sorry :D